guide/onboarding/automatic-cloud-backup.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: guide
+title: Automatic cloud backup
+description: How to backup a recovery phrase automatically
+grand_parent: Onboarding
+parent: Backing up a recovery phrase
+nav_order: 1
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/
+main_classes: -no-top-padding
+---
+
+# Automatic cloud backup
+
+### How it works
+
+The user’s key is encrypted and backed up on their respective device/cloud storage provider. If their device is lost, stolen, or broken, the user can log into their cloud storage account (such as Apple iCloud or Google Drive) to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
+
+> Crypto rookies perceive themselves as “incapable of self-managing keys”.
+>
+> <cite>As researched by <a href="https://voskart.de/pdf/u_in_crypto.pdf">Abramova et al</a></cite>
+
+This method provides users with the feeling that they can rely on their cloud provider to regain access to their funds rather than putting the responsibility on themselves.
+
+Below is an example of what the user interface might look like for an automatic cloud backup that significantly reduces friction during onboarding.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro@2x.png"
+   layout = "shadow"
+   alt-text = "Screen explaining automatic cloud backups"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+     image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-password.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-password@2x.png"
+   class = "shadow"
+   alt-text = "The user can set a password which will be used to encrypt their recovery phrase and act as a login"
+   width = 250
+   height = 541
+%}
```

Comment:
I am struggling with this. On the one hand, it makes absolute sense that you want your recovery seed to be password protected. On the other hand, I don't feel like this solves any problems for a new user. How is this any different from 24 seed words? If they are capable of losing their seed words, it seems to me that they are equally as likely to lose the passphrase which encrypts their seed words.

I think there are ways around this (Casa has some scheme for it I believe).

I don't intend to over-complicate this, it just seems to me like a recovery phrase and a passphrase are different ways of describing the same thing: something that a new user will ultimately lose.

Is there a detail I am missing?

Reply:
This is a good point.

In practice rather than using a passphrase or password, a simple 4 to 6 digit PIN would be sufficient to gain access to the wallet, it's easier to remember and the user is less likely to worry about having to back this up. 

In practice, this is what we have been experimenting with on the [Photon SDK ](https://github.com/photon-sdk/photon-lib) and it works well. 

I'll replace the images of the password screen with a PIN-based one as this is more in line with what we would want to advocate for. Keep in mind this scheme is for what a user would consider low-value amounts.